ui/src/components/WorkersTable/index.jsx

```diff
@@ -114,7 +125,8 @@ export default class WorkersTable extends Component {
   }
 
   render() {
-    const { sortBy, sortDirection } = this.state;
+    const query = parse(window.location.search.slice(1));
+    const { sortBy, sortDirection } = query.sortBy ? query : this.state;
```

Comment:
Now that we are reading the sort from the URL, there is no need to read from `this.state`. Let's remove `sortBy` and `sortDirection` from the `state` object. Please apply this change for the other tables in this PR.